I, [2013-11-15T21:25:51.642029 #28303]  INFO -- : Started GET "/welcome/index" for 127.0.0.1 at 2013-11-15 21:25:51 -0600
F, [2013-11-15T21:25:51.651582 #28303] FATAL -- : 
PG::ConnectionBad (FATAL:  database "hack_tx_production" does not exist
):
  activerecord (4.0.1) lib/active_record/connection_adapters/postgresql_adapter.rb:831:in `initialize'
  activerecord (4.0.1) lib/active_record/connection_adapters/postgresql_adapter.rb:831:in `new'
  activerecord (4.0.1) lib/active_record/connection_adapters/postgresql_adapter.rb:831:in `connect'
  activerecord (4.0.1) lib/active_record/connection_adapters/postgresql_adapter.rb:548:in `initialize'
  activerecord (4.0.1) lib/active_record/connection_adapters/postgresql_adapter.rb:41:in `new'
  activerecord (4.0.1) lib/active_record/connection_adapters/postgresql_adapter.rb:41:in `postgresql_connection'
  activerecord (4.0.1) lib/active_record/connection_adapters/abstract/connection_pool.rb:440:in `new_connection'
  activerecord (4.0.1) lib/active_record/connection_adapters/abstract/connection_pool.rb:450:in `checkout_new_connection'
  activerecord (4.0.1) lib/active_record/connection_adapters/abstract/connection_pool.rb:421:in `acquire_connection'
  activerecord (4.0.1) lib/active_record/connection_adapters/abstract/connection_pool.rb:356:in `block in checkout'
  /Users/katie/.rvm/rubies/ruby-2.0.0-p247/lib/ruby/2.0.0/monitor.rb:211:in `mon_synchronize'
  activerecord (4.0.1) lib/active_record/connection_adapters/abstract/connection_pool.rb:355:in `checkout'
  activerecord (4.0.1) lib/active_record/connection_adapters/abstract/connection_pool.rb:265:in `block in connection'
  /Users/katie/.rvm/rubies/ruby-2.0.0-p247/lib/ruby/2.0.0/monitor.rb:211:in `mon_synchronize'
  activerecord (4.0.1) lib/active_record/connection_adapters/abstract/connection_pool.rb:264:in `connection'
  activerecord (4.0.1) lib/active_record/connection_adapters/abstract/connection_pool.rb:546:in `retrieve_connection'
  activerecord (4.0.1) lib/active_record/connection_handling.rb:79:in `retrieve_connection'
  activerecord (4.0.1) lib/active_record/connection_handling.rb:53:in `connection'
  activerecord (4.0.1) lib/active_record/query_cache.rb:51:in `restore_query_cache_settings'
  activerecord (4.0.1) lib/active_record/query_cache.rb:43:in `rescue in call'
  activerecord (4.0.1) lib/active_record/query_cache.rb:32:in `call'
  activerecord (4.0.1) lib/active_record/connection_adapters/abstract/connection_pool.rb:626:in `call'
  actionpack (4.0.1) lib/action_dispatch/middleware/callbacks.rb:29:in `block in call'
  activesupport (4.0.1) lib/active_support/callbacks.rb:373:in `_run__3139354468937840955__call__callbacks'
  activesupport (4.0.1) lib/active_support/callbacks.rb:80:in `run_callbacks'
  actionpack (4.0.1) lib/action_dispatch/middleware/callbacks.rb:27:in `call'
  actionpack (4.0.1) lib/action_dispatch/middleware/remote_ip.rb:76:in `call'
  actionpack (4.0.1) lib/action_dispatch/middleware/debug_exceptions.rb:17:in `call'
  actionpack (4.0.1) lib/action_dispatch/middleware/show_exceptions.rb:30:in `call'
  railties (4.0.1) lib/rails/rack/logger.rb:38:in `call_app'
  railties (4.0.1) lib/rails/rack/logger.rb:20:in `block in call'
  activesupport (4.0.1) lib/active_support/tagged_logging.rb:67:in `block in tagged'
  activesupport (4.0.1) lib/active_support/tagged_logging.rb:25:in `tagged'
  activesupport (4.0.1) lib/active_support/tagged_logging.rb:67:in `tagged'
  railties (4.0.1) lib/rails/rack/logger.rb:20:in `call'
  actionpack (4.0.1) lib/action_dispatch/middleware/request_id.rb:21:in `call'
  rack (1.5.2) lib/rack/methodoverride.rb:21:in `call'
  rack (1.5.2) lib/rack/runtime.rb:17:in `call'
  activesupport (4.0.1) lib/active_support/cache/strategy/local_cache.rb:83:in `call'
  rack (1.5.2) lib/rack/sendfile.rb:112:in `call'
  railties (4.0.1) lib/rails/engine.rb:511:in `call'
  railties (4.0.1) lib/rails/application.rb:97:in `call'
  rack (1.5.2) lib/rack/lock.rb:17:in `call'
  rack (1.5.2) lib/rack/content_length.rb:14:in `call'
  rack (1.5.2) lib/rack/handler/webrick.rb:60:in `service'
  /Users/katie/.rvm/rubies/ruby-2.0.0-p247/lib/ruby/2.0.0/webrick/httpserver.rb:138:in `service'
  /Users/katie/.rvm/rubies/ruby-2.0.0-p247/lib/ruby/2.0.0/webrick/httpserver.rb:94:in `run'
  /Users/katie/.rvm/rubies/ruby-2.0.0-p247/lib/ruby/2.0.0/webrick/server.rb:295:in `block in start_thread'


I, [2013-11-15T21:25:53.220011 #28303]  INFO -- : Started GET "/welcome/index" for 127.0.0.1 at 2013-11-15 21:25:53 -0600
F, [2013-11-15T21:25:53.227853 #28303] FATAL -- : 
PG::ConnectionBad (FATAL:  database "hack_tx_production" does not exist
):
  activerecord (4.0.1) lib/active_record/connection_adapters/postgresql_adapter.rb:831:in `initialize'
  activerecord (4.0.1) lib/active_record/connection_adapters/postgresql_adapter.rb:831:in `new'
  activerecord (4.0.1) lib/active_record/connection_adapters/postgresql_adapter.rb:831:in `connect'
  activerecord (4.0.1) lib/active_record/connection_adapters/postgresql_adapter.rb:548:in `initialize'
  activerecord (4.0.1) lib/active_record/connection_adapters/postgresql_adapter.rb:41:in `new'
  activerecord (4.0.1) lib/active_record/connection_adapters/postgresql_adapter.rb:41:in `postgresql_connection'
  activerecord (4.0.1) lib/active_record/connection_adapters/abstract/connection_pool.rb:440:in `new_connection'
  activerecord (4.0.1) lib/active_record/connection_adapters/abstract/connection_pool.rb:450:in `checkout_new_connection'
  activerecord (4.0.1) lib/active_record/connection_adapters/abstract/connection_pool.rb:421:in `acquire_connection'
  activerecord (4.0.1) lib/active_record/connection_adapters/abstract/connection_pool.rb:356:in `block in checkout'
  /Users/katie/.rvm/rubies/ruby-2.0.0-p247/lib/ruby/2.0.0/monitor.rb:211:in `mon_synchronize'
  activerecord (4.0.1) lib/active_record/connection_adapters/abstract/connection_pool.rb:355:in `checkout'
  activerecord (4.0.1) lib/active_record/connection_adapters/abstract/connection_pool.rb:265:in `block in connection'
  /Users/katie/.rvm/rubies/ruby-2.0.0-p247/lib/ruby/2.0.0/monitor.rb:211:in `mon_synchronize'
  activerecord (4.0.1) lib/active_record/connection_adapters/abstract/connection_pool.rb:264:in `connection'
  activerecord (4.0.1) lib/active_record/connection_adapters/abstract/connection_pool.rb:546:in `retrieve_connection'
  activerecord (4.0.1) lib/active_record/connection_handling.rb:79:in `retrieve_connection'
  activerecord (4.0.1) lib/active_record/connection_handling.rb:53:in `connection'
  activerecord (4.0.1) lib/active_record/query_cache.rb:51:in `restore_query_cache_settings'
  activerecord (4.0.1) lib/active_record/query_cache.rb:43:in `rescue in call'
  activerecord (4.0.1) lib/active_record/query_cache.rb:32:in `call'
  activerecord (4.0.1) lib/active_record/connection_adapters/abstract/connection_pool.rb:626:in `call'
  actionpack (4.0.1) lib/action_dispatch/middleware/callbacks.rb:29:in `block in call'
  activesupport (4.0.1) lib/active_support/callbacks.rb:373:in `_run__3139354468937840955__call__callbacks'
  activesupport (4.0.1) lib/active_support/callbacks.rb:80:in `run_callbacks'
  actionpack (4.0.1) lib/action_dispatch/middleware/callbacks.rb:27:in `call'
  actionpack (4.0.1) lib/action_dispatch/middleware/remote_ip.rb:76:in `call'
  actionpack (4.0.1) lib/action_dispatch/middleware/debug_exceptions.rb:17:in `call'
  actionpack (4.0.1) lib/action_dispatch/middleware/show_exceptions.rb:30:in `call'
  railties (4.0.1) lib/rails/rack/logger.rb:38:in `call_app'
  railties (4.0.1) lib/rails/rack/logger.rb:20:in `block in call'
  activesupport (4.0.1) lib/active_support/tagged_logging.rb:67:in `block in tagged'
  activesupport (4.0.1) lib/active_support/tagged_logging.rb:25:in `tagged'
  activesupport (4.0.1) lib/active_support/tagged_logging.rb:67:in `tagged'
  railties (4.0.1) lib/rails/rack/logger.rb:20:in `call'
  actionpack (4.0.1) lib/action_dispatch/middleware/request_id.rb:21:in `call'
  rack (1.5.2) lib/rack/methodoverride.rb:21:in `call'
  rack (1.5.2) lib/rack/runtime.rb:17:in `call'
  activesupport (4.0.1) lib/active_support/cache/strategy/local_cache.rb:83:in `call'
  rack (1.5.2) lib/rack/sendfile.rb:112:in `call'
  railties (4.0.1) lib/rails/engine.rb:511:in `call'
  railties (4.0.1) lib/rails/application.rb:97:in `call'
  rack (1.5.2) lib/rack/lock.rb:17:in `call'
  rack (1.5.2) lib/rack/content_length.rb:14:in `call'
  rack (1.5.2) lib/rack/handler/webrick.rb:60:in `service'
  /Users/katie/.rvm/rubies/ruby-2.0.0-p247/lib/ruby/2.0.0/webrick/httpserver.rb:138:in `service'
  /Users/katie/.rvm/rubies/ruby-2.0.0-p247/lib/ruby/2.0.0/webrick/httpserver.rb:94:in `run'
  /Users/katie/.rvm/rubies/ruby-2.0.0-p247/lib/ruby/2.0.0/webrick/server.rb:295:in `block in start_thread'


I, [2013-11-15T21:25:59.489495 #28303]  INFO -- : Started GET "/" for 127.0.0.1 at 2013-11-15 21:25:59 -0600
F, [2013-11-15T21:25:59.498031 #28303] FATAL -- : 
PG::ConnectionBad (FATAL:  database "hack_tx_production" does not exist
):
  activerecord (4.0.1) lib/active_record/connection_adapters/postgresql_adapter.rb:831:in `initialize'
  activerecord (4.0.1) lib/active_record/connection_adapters/postgresql_adapter.rb:831:in `new'
  activerecord (4.0.1) lib/active_record/connection_adapters/postgresql_adapter.rb:831:in `connect'
  activerecord (4.0.1) lib/active_record/connection_adapters/postgresql_adapter.rb:548:in `initialize'
  activerecord (4.0.1) lib/active_record/connection_adapters/postgresql_adapter.rb:41:in `new'
  activerecord (4.0.1) lib/active_record/connection_adapters/postgresql_adapter.rb:41:in `postgresql_connection'
  activerecord (4.0.1) lib/active_record/connection_adapters/abstract/connection_pool.rb:440:in `new_connection'
  activerecord (4.0.1) lib/active_record/connection_adapters/abstract/connection_pool.rb:450:in `checkout_new_connection'
  activerecord (4.0.1) lib/active_record/connection_adapters/abstract/connection_pool.rb:421:in `acquire_connection'
  activerecord (4.0.1) lib/active_record/connection_adapters/abstract/connection_pool.rb:356:in `block in checkout'
  /Users/katie/.rvm/rubies/ruby-2.0.0-p247/lib/ruby/2.0.0/monitor.rb:211:in `mon_synchronize'
  activerecord (4.0.1) lib/active_record/connection_adapters/abstract/connection_pool.rb:355:in `checkout'
  activerecord (4.0.1) lib/active_record/connection_adapters/abstract/connection_pool.rb:265:in `block in connection'
  /Users/katie/.rvm/rubies/ruby-2.0.0-p247/lib/ruby/2.0.0/monitor.rb:211:in `mon_synchronize'
  activerecord (4.0.1) lib/active_record/connection_adapters/abstract/connection_pool.rb:264:in `connection'
  activerecord (4.0.1) lib/active_record/connection_adapters/abstract/connection_pool.rb:546:in `retrieve_connection'
  activerecord (4.0.1) lib/active_record/connection_handling.rb:79:in `retrieve_connection'
  activerecord (4.0.1) lib/active_record/connection_handling.rb:53:in `connection'
  activerecord (4.0.1) lib/active_record/query_cache.rb:51:in `restore_query_cache_settings'
  activerecord (4.0.1) lib/active_record/query_cache.rb:43:in `rescue in call'
  activerecord (4.0.1) lib/active_record/query_cache.rb:32:in `call'
  activerecord (4.0.1) lib/active_record/connection_adapters/abstract/connection_pool.rb:626:in `call'
  actionpack (4.0.1) lib/action_dispatch/middleware/callbacks.rb:29:in `block in call'
  activesupport (4.0.1) lib/active_support/callbacks.rb:373:in `_run__3139354468937840955__call__callbacks'
  activesupport (4.0.1) lib/active_support/callbacks.rb:80:in `run_callbacks'
  actionpack (4.0.1) lib/action_dispatch/middleware/callbacks.rb:27:in `call'
  actionpack (4.0.1) lib/action_dispatch/middleware/remote_ip.rb:76:in `call'
  actionpack (4.0.1) lib/action_dispatch/middleware/debug_exceptions.rb:17:in `call'
  actionpack (4.0.1) lib/action_dispatch/middleware/show_exceptions.rb:30:in `call'
  railties (4.0.1) lib/rails/rack/logger.rb:38:in `call_app'
  railties (4.0.1) lib/rails/rack/logger.rb:20:in `block in call'
  activesupport (4.0.1) lib/active_support/tagged_logging.rb:67:in `block in tagged'
  activesupport (4.0.1) lib/active_support/tagged_logging.rb:25:in `tagged'
  activesupport (4.0.1) lib/active_support/tagged_logging.rb:67:in `tagged'
  railties (4.0.1) lib/rails/rack/logger.rb:20:in `call'
  actionpack (4.0.1) lib/action_dispatch/middleware/request_id.rb:21:in `call'
  rack (1.5.2) lib/rack/methodoverride.rb:21:in `call'
  rack (1.5.2) lib/rack/runtime.rb:17:in `call'
  activesupport (4.0.1) lib/active_support/cache/strategy/local_cache.rb:83:in `call'
  rack (1.5.2) lib/rack/sendfile.rb:112:in `call'
  railties (4.0.1) lib/rails/engine.rb:511:in `call'
  railties (4.0.1) lib/rails/application.rb:97:in `call'
  rack (1.5.2) lib/rack/lock.rb:17:in `call'
  rack (1.5.2) lib/rack/content_length.rb:14:in `call'
  rack (1.5.2) lib/rack/handler/webrick.rb:60:in `service'
  /Users/katie/.rvm/rubies/ruby-2.0.0-p247/lib/ruby/2.0.0/webrick/httpserver.rb:138:in `service'
  /Users/katie/.rvm/rubies/ruby-2.0.0-p247/lib/ruby/2.0.0/webrick/httpserver.rb:94:in `run'
  /Users/katie/.rvm/rubies/ruby-2.0.0-p247/lib/ruby/2.0.0/webrick/server.rb:295:in `block in start_thread'


I, [2013-11-15T21:27:56.733129 #28303]  INFO -- : Started GET "/" for 127.0.0.1 at 2013-11-15 21:27:56 -0600
F, [2013-11-15T21:27:56.740586 #28303] FATAL -- : 
PG::ConnectionBad (FATAL:  database "hack_tx_production" does not exist
):
  activerecord (4.0.1) lib/active_record/connection_adapters/postgresql_adapter.rb:831:in `initialize'
  activerecord (4.0.1) lib/active_record/connection_adapters/postgresql_adapter.rb:831:in `new'
  activerecord (4.0.1) lib/active_record/connection_adapters/postgresql_adapter.rb:831:in `connect'
  activerecord (4.0.1) lib/active_record/connection_adapters/postgresql_adapter.rb:548:in `initialize'
  activerecord (4.0.1) lib/active_record/connection_adapters/postgresql_adapter.rb:41:in `new'
  activerecord (4.0.1) lib/active_record/connection_adapters/postgresql_adapter.rb:41:in `postgresql_connection'
  activerecord (4.0.1) lib/active_record/connection_adapters/abstract/connection_pool.rb:440:in `new_connection'
  activerecord (4.0.1) lib/active_record/connection_adapters/abstract/connection_pool.rb:450:in `checkout_new_connection'
  activerecord (4.0.1) lib/active_record/connection_adapters/abstract/connection_pool.rb:421:in `acquire_connection'
  activerecord (4.0.1) lib/active_record/connection_adapters/abstract/connection_pool.rb:356:in `block in checkout'
  /Users/katie/.rvm/rubies/ruby-2.0.0-p247/lib/ruby/2.0.0/monitor.rb:211:in `mon_synchronize'
  activerecord (4.0.1) lib/active_record/connection_adapters/abstract/connection_pool.rb:355:in `checkout'
  activerecord (4.0.1) lib/active_record/connection_adapters/abstract/connection_pool.rb:265:in `block in connection'
  /Users/katie/.rvm/rubies/ruby-2.0.0-p247/lib/ruby/2.0.0/monitor.rb:211:in `mon_synchronize'
  activerecord (4.0.1) lib/active_record/connection_adapters/abstract/connection_pool.rb:264:in `connection'
  activerecord (4.0.1) lib/active_record/connection_adapters/abstract/connection_pool.rb:546:in `retrieve_connection'
  activerecord (4.0.1) lib/active_record/connection_handling.rb:79:in `retrieve_connection'
  activerecord (4.0.1) lib/active_record/connection_handling.rb:53:in `connection'
  activerecord (4.0.1) lib/active_record/query_cache.rb:51:in `restore_query_cache_settings'
  activerecord (4.0.1) lib/active_record/query_cache.rb:43:in `rescue in call'
  activerecord (4.0.1) lib/active_record/query_cache.rb:32:in `call'
  activerecord (4.0.1) lib/active_record/connection_adapters/abstract/connection_pool.rb:626:in `call'
  actionpack (4.0.1) lib/action_dispatch/middleware/callbacks.rb:29:in `block in call'
  activesupport (4.0.1) lib/active_support/callbacks.rb:373:in `_run__3139354468937840955__call__callbacks'
  activesupport (4.0.1) lib/active_support/callbacks.rb:80:in `run_callbacks'
  actionpack (4.0.1) lib/action_dispatch/middleware/callbacks.rb:27:in `call'
  actionpack (4.0.1) lib/action_dispatch/middleware/remote_ip.rb:76:in `call'
  actionpack (4.0.1) lib/action_dispatch/middleware/debug_exceptions.rb:17:in `call'
  actionpack (4.0.1) lib/action_dispatch/middleware/show_exceptions.rb:30:in `call'
  railties (4.0.1) lib/rails/rack/logger.rb:38:in `call_app'
  railties (4.0.1) lib/rails/rack/logger.rb:20:in `block in call'
  activesupport (4.0.1) lib/active_support/tagged_logging.rb:67:in `block in tagged'
  activesupport (4.0.1) lib/active_support/tagged_logging.rb:25:in `tagged'
  activesupport (4.0.1) lib/active_support/tagged_logging.rb:67:in `tagged'
  railties (4.0.1) lib/rails/rack/logger.rb:20:in `call'
  actionpack (4.0.1) lib/action_dispatch/middleware/request_id.rb:21:in `call'
  rack (1.5.2) lib/rack/methodoverride.rb:21:in `call'
  rack (1.5.2) lib/rack/runtime.rb:17:in `call'
  activesupport (4.0.1) lib/active_support/cache/strategy/local_cache.rb:83:in `call'
  rack (1.5.2) lib/rack/sendfile.rb:112:in `call'
  railties (4.0.1) lib/rails/engine.rb:511:in `call'
  railties (4.0.1) lib/rails/application.rb:97:in `call'
  rack (1.5.2) lib/rack/lock.rb:17:in `call'
  rack (1.5.2) lib/rack/content_length.rb:14:in `call'
  rack (1.5.2) lib/rack/handler/webrick.rb:60:in `service'
  /Users/katie/.rvm/rubies/ruby-2.0.0-p247/lib/ruby/2.0.0/webrick/httpserver.rb:138:in `service'
  /Users/katie/.rvm/rubies/ruby-2.0.0-p247/lib/ruby/2.0.0/webrick/httpserver.rb:94:in `run'
  /Users/katie/.rvm/rubies/ruby-2.0.0-p247/lib/ruby/2.0.0/webrick/server.rb:295:in `block in start_thread'


I, [2013-11-15T21:27:58.853950 #28303]  INFO -- : Started GET "/" for 127.0.0.1 at 2013-11-15 21:27:58 -0600
F, [2013-11-15T21:27:58.861321 #28303] FATAL -- : 
PG::ConnectionBad (FATAL:  database "hack_tx_production" does not exist
):
  activerecord (4.0.1) lib/active_record/connection_adapters/postgresql_adapter.rb:831:in `initialize'
  activerecord (4.0.1) lib/active_record/connection_adapters/postgresql_adapter.rb:831:in `new'
  activerecord (4.0.1) lib/active_record/connection_adapters/postgresql_adapter.rb:831:in `connect'
  activerecord (4.0.1) lib/active_record/connection_adapters/postgresql_adapter.rb:548:in `initialize'
  activerecord (4.0.1) lib/active_record/connection_adapters/postgresql_adapter.rb:41:in `new'
  activerecord (4.0.1) lib/active_record/connection_adapters/postgresql_adapter.rb:41:in `postgresql_connection'
  activerecord (4.0.1) lib/active_record/connection_adapters/abstract/connection_pool.rb:440:in `new_connection'
  activerecord (4.0.1) lib/active_record/connection_adapters/abstract/connection_pool.rb:450:in `checkout_new_connection'
  activerecord (4.0.1) lib/active_record/connection_adapters/abstract/connection_pool.rb:421:in `acquire_connection'
  activerecord (4.0.1) lib/active_record/connection_adapters/abstract/connection_pool.rb:356:in `block in checkout'
  /Users/katie/.rvm/rubies/ruby-2.0.0-p247/lib/ruby/2.0.0/monitor.rb:211:in `mon_synchronize'
  activerecord (4.0.1) lib/active_record/connection_adapters/abstract/connection_pool.rb:355:in `checkout'
  activerecord (4.0.1) lib/active_record/connection_adapters/abstract/connection_pool.rb:265:in `block in connection'
  /Users/katie/.rvm/rubies/ruby-2.0.0-p247/lib/ruby/2.0.0/monitor.rb:211:in `mon_synchronize'
  activerecord (4.0.1) lib/active_record/connection_adapters/abstract/connection_pool.rb:264:in `connection'
  activerecord (4.0.1) lib/active_record/connection_adapters/abstract/connection_pool.rb:546:in `retrieve_connection'
  activerecord (4.0.1) lib/active_record/connection_handling.rb:79:in `retrieve_connection'
  activerecord (4.0.1) lib/active_record/connection_handling.rb:53:in `connection'
  activerecord (4.0.1) lib/active_record/query_cache.rb:51:in `restore_query_cache_settings'
  activerecord (4.0.1) lib/active_record/query_cache.rb:43:in `rescue in call'
  activerecord (4.0.1) lib/active_record/query_cache.rb:32:in `call'
  activerecord (4.0.1) lib/active_record/connection_adapters/abstract/connection_pool.rb:626:in `call'
  actionpack (4.0.1) lib/action_dispatch/middleware/callbacks.rb:29:in `block in call'
  activesupport (4.0.1) lib/active_support/callbacks.rb:373:in `_run__3139354468937840955__call__callbacks'
  activesupport (4.0.1) lib/active_support/callbacks.rb:80:in `run_callbacks'
  actionpack (4.0.1) lib/action_dispatch/middleware/callbacks.rb:27:in `call'
  actionpack (4.0.1) lib/action_dispatch/middleware/remote_ip.rb:76:in `call'
  actionpack (4.0.1) lib/action_dispatch/middleware/debug_exceptions.rb:17:in `call'
  actionpack (4.0.1) lib/action_dispatch/middleware/show_exceptions.rb:30:in `call'
  railties (4.0.1) lib/rails/rack/logger.rb:38:in `call_app'
  railties (4.0.1) lib/rails/rack/logger.rb:20:in `block in call'
  activesupport (4.0.1) lib/active_support/tagged_logging.rb:67:in `block in tagged'
  activesupport (4.0.1) lib/active_support/tagged_logging.rb:25:in `tagged'
  activesupport (4.0.1) lib/active_support/tagged_logging.rb:67:in `tagged'
  railties (4.0.1) lib/rails/rack/logger.rb:20:in `call'
  actionpack (4.0.1) lib/action_dispatch/middleware/request_id.rb:21:in `call'
  rack (1.5.2) lib/rack/methodoverride.rb:21:in `call'
  rack (1.5.2) lib/rack/runtime.rb:17:in `call'
  activesupport (4.0.1) lib/active_support/cache/strategy/local_cache.rb:83:in `call'
  rack (1.5.2) lib/rack/sendfile.rb:112:in `call'
  railties (4.0.1) lib/rails/engine.rb:511:in `call'
  railties (4.0.1) lib/rails/application.rb:97:in `call'
  rack (1.5.2) lib/rack/lock.rb:17:in `call'
  rack (1.5.2) lib/rack/content_length.rb:14:in `call'
  rack (1.5.2) lib/rack/handler/webrick.rb:60:in `service'
  /Users/katie/.rvm/rubies/ruby-2.0.0-p247/lib/ruby/2.0.0/webrick/httpserver.rb:138:in `service'
  /Users/katie/.rvm/rubies/ruby-2.0.0-p247/lib/ruby/2.0.0/webrick/httpserver.rb:94:in `run'
  /Users/katie/.rvm/rubies/ruby-2.0.0-p247/lib/ruby/2.0.0/webrick/server.rb:295:in `block in start_thread'


I, [2013-11-15T21:28:12.248157 #28303]  INFO -- : Started GET "/" for 127.0.0.1 at 2013-11-15 21:28:12 -0600
F, [2013-11-15T21:28:12.256381 #28303] FATAL -- : 
PG::ConnectionBad (FATAL:  database "hack_tx_production" does not exist
):
  activerecord (4.0.1) lib/active_record/connection_adapters/postgresql_adapter.rb:831:in `initialize'
  activerecord (4.0.1) lib/active_record/connection_adapters/postgresql_adapter.rb:831:in `new'
  activerecord (4.0.1) lib/active_record/connection_adapters/postgresql_adapter.rb:831:in `connect'
  activerecord (4.0.1) lib/active_record/connection_adapters/postgresql_adapter.rb:548:in `initialize'
  activerecord (4.0.1) lib/active_record/connection_adapters/postgresql_adapter.rb:41:in `new'
  activerecord (4.0.1) lib/active_record/connection_adapters/postgresql_adapter.rb:41:in `postgresql_connection'
  activerecord (4.0.1) lib/active_record/connection_adapters/abstract/connection_pool.rb:440:in `new_connection'
  activerecord (4.0.1) lib/active_record/connection_adapters/abstract/connection_pool.rb:450:in `checkout_new_connection'
  activerecord (4.0.1) lib/active_record/connection_adapters/abstract/connection_pool.rb:421:in `acquire_connection'
  activerecord (4.0.1) lib/active_record/connection_adapters/abstract/connection_pool.rb:356:in `block in checkout'
  /Users/katie/.rvm/rubies/ruby-2.0.0-p247/lib/ruby/2.0.0/monitor.rb:211:in `mon_synchronize'
  activerecord (4.0.1) lib/active_record/connection_adapters/abstract/connection_pool.rb:355:in `checkout'
  activerecord (4.0.1) lib/active_record/connection_adapters/abstract/connection_pool.rb:265:in `block in connection'
  /Users/katie/.rvm/rubies/ruby-2.0.0-p247/lib/ruby/2.0.0/monitor.rb:211:in `mon_synchronize'
  activerecord (4.0.1) lib/active_record/connection_adapters/abstract/connection_pool.rb:264:in `connection'
  activerecord (4.0.1) lib/active_record/connection_adapters/abstract/connection_pool.rb:546:in `retrieve_connection'
  activerecord (4.0.1) lib/active_record/connection_handling.rb:79:in `retrieve_connection'
  activerecord (4.0.1) lib/active_record/connection_handling.rb:53:in `connection'
  activerecord (4.0.1) lib/active_record/query_cache.rb:51:in `restore_query_cache_settings'
  activerecord (4.0.1) lib/active_record/query_cache.rb:43:in `rescue in call'
  activerecord (4.0.1) lib/active_record/query_cache.rb:32:in `call'
  activerecord (4.0.1) lib/active_record/connection_adapters/abstract/connection_pool.rb:626:in `call'
  actionpack (4.0.1) lib/action_dispatch/middleware/callbacks.rb:29:in `block in call'
  activesupport (4.0.1) lib/active_support/callbacks.rb:373:in `_run__3139354468937840955__call__callbacks'
  activesupport (4.0.1) lib/active_support/callbacks.rb:80:in `run_callbacks'
  actionpack (4.0.1) lib/action_dispatch/middleware/callbacks.rb:27:in `call'
  actionpack (4.0.1) lib/action_dispatch/middleware/remote_ip.rb:76:in `call'
  actionpack (4.0.1) lib/action_dispatch/middleware/debug_exceptions.rb:17:in `call'
  actionpack (4.0.1) lib/action_dispatch/middleware/show_exceptions.rb:30:in `call'
  railties (4.0.1) lib/rails/rack/logger.rb:38:in `call_app'
  railties (4.0.1) lib/rails/rack/logger.rb:20:in `block in call'
  activesupport (4.0.1) lib/active_support/tagged_logging.rb:67:in `block in tagged'
  activesupport (4.0.1) lib/active_support/tagged_logging.rb:25:in `tagged'
  activesupport (4.0.1) lib/active_support/tagged_logging.rb:67:in `tagged'
  railties (4.0.1) lib/rails/rack/logger.rb:20:in `call'
  actionpack (4.0.1) lib/action_dispatch/middleware/request_id.rb:21:in `call'
  rack (1.5.2) lib/rack/methodoverride.rb:21:in `call'
  rack (1.5.2) lib/rack/runtime.rb:17:in `call'
  activesupport (4.0.1) lib/active_support/cache/strategy/local_cache.rb:83:in `call'
  rack (1.5.2) lib/rack/sendfile.rb:112:in `call'
  railties (4.0.1) lib/rails/engine.rb:511:in `call'
  railties (4.0.1) lib/rails/application.rb:97:in `call'
  rack (1.5.2) lib/rack/lock.rb:17:in `call'
  rack (1.5.2) lib/rack/content_length.rb:14:in `call'
  rack (1.5.2) lib/rack/handler/webrick.rb:60:in `service'
  /Users/katie/.rvm/rubies/ruby-2.0.0-p247/lib/ruby/2.0.0/webrick/httpserver.rb:138:in `service'
  /Users/katie/.rvm/rubies/ruby-2.0.0-p247/lib/ruby/2.0.0/webrick/httpserver.rb:94:in `run'
  /Users/katie/.rvm/rubies/ruby-2.0.0-p247/lib/ruby/2.0.0/webrick/server.rb:295:in `block in start_thread'


I, [2013-11-15T21:30:56.853367 #28303]  INFO -- : Started GET "/" for 127.0.0.1 at 2013-11-15 21:30:56 -0600
F, [2013-11-15T21:30:56.861099 #28303] FATAL -- : 
PG::ConnectionBad (FATAL:  database "hack_tx_production" does not exist
):
  activerecord (4.0.1) lib/active_record/connection_adapters/postgresql_adapter.rb:831:in `initialize'
  activerecord (4.0.1) lib/active_record/connection_adapters/postgresql_adapter.rb:831:in `new'
  activerecord (4.0.1) lib/active_record/connection_adapters/postgresql_adapter.rb:831:in `connect'
  activerecord (4.0.1) lib/active_record/connection_adapters/postgresql_adapter.rb:548:in `initialize'
  activerecord (4.0.1) lib/active_record/connection_adapters/postgresql_adapter.rb:41:in `new'
  activerecord (4.0.1) lib/active_record/connection_adapters/postgresql_adapter.rb:41:in `postgresql_connection'
  activerecord (4.0.1) lib/active_record/connection_adapters/abstract/connection_pool.rb:440:in `new_connection'
  activerecord (4.0.1) lib/active_record/connection_adapters/abstract/connection_pool.rb:450:in `checkout_new_connection'
  activerecord (4.0.1) lib/active_record/connection_adapters/abstract/connection_pool.rb:421:in `acquire_connection'
  activerecord (4.0.1) lib/active_record/connection_adapters/abstract/connection_pool.rb:356:in `block in checkout'
  /Users/katie/.rvm/rubies/ruby-2.0.0-p247/lib/ruby/2.0.0/monitor.rb:211:in `mon_synchronize'
  activerecord (4.0.1) lib/active_record/connection_adapters/abstract/connection_pool.rb:355:in `checkout'
  activerecord (4.0.1) lib/active_record/connection_adapters/abstract/connection_pool.rb:265:in `block in connection'
  /Users/katie/.rvm/rubies/ruby-2.0.0-p247/lib/ruby/2.0.0/monitor.rb:211:in `mon_synchronize'
  activerecord (4.0.1) lib/active_record/connection_adapters/abstract/connection_pool.rb:264:in `connection'
  activerecord (4.0.1) lib/active_record/connection_adapters/abstract/connection_pool.rb:546:in `retrieve_connection'
  activerecord (4.0.1) lib/active_record/connection_handling.rb:79:in `retrieve_connection'
  activerecord (4.0.1) lib/active_record/connection_handling.rb:53:in `connection'
  activerecord (4.0.1) lib/active_record/query_cache.rb:51:in `restore_query_cache_settings'
  activerecord (4.0.1) lib/active_record/query_cache.rb:43:in `rescue in call'
  activerecord (4.0.1) lib/active_record/query_cache.rb:32:in `call'
  activerecord (4.0.1) lib/active_record/connection_adapters/abstract/connection_pool.rb:626:in `call'
  actionpack (4.0.1) lib/action_dispatch/middleware/callbacks.rb:29:in `block in call'
  activesupport (4.0.1) lib/active_support/callbacks.rb:373:in `_run__3139354468937840955__call__callbacks'
  activesupport (4.0.1) lib/active_support/callbacks.rb:80:in `run_callbacks'
  actionpack (4.0.1) lib/action_dispatch/middleware/callbacks.rb:27:in `call'
  actionpack (4.0.1) lib/action_dispatch/middleware/remote_ip.rb:76:in `call'
  actionpack (4.0.1) lib/action_dispatch/middleware/debug_exceptions.rb:17:in `call'
  actionpack (4.0.1) lib/action_dispatch/middleware/show_exceptions.rb:30:in `call'
  railties (4.0.1) lib/rails/rack/logger.rb:38:in `call_app'
  railties (4.0.1) lib/rails/rack/logger.rb:20:in `block in call'
  activesupport (4.0.1) lib/active_support/tagged_logging.rb:67:in `block in tagged'
  activesupport (4.0.1) lib/active_support/tagged_logging.rb:25:in `tagged'
  activesupport (4.0.1) lib/active_support/tagged_logging.rb:67:in `tagged'
  railties (4.0.1) lib/rails/rack/logger.rb:20:in `call'
  actionpack (4.0.1) lib/action_dispatch/middleware/request_id.rb:21:in `call'
  rack (1.5.2) lib/rack/methodoverride.rb:21:in `call'
  rack (1.5.2) lib/rack/runtime.rb:17:in `call'
  activesupport (4.0.1) lib/active_support/cache/strategy/local_cache.rb:83:in `call'
  rack (1.5.2) lib/rack/sendfile.rb:112:in `call'
  railties (4.0.1) lib/rails/engine.rb:511:in `call'
  railties (4.0.1) lib/rails/application.rb:97:in `call'
  rack (1.5.2) lib/rack/lock.rb:17:in `call'
  rack (1.5.2) lib/rack/content_length.rb:14:in `call'
  rack (1.5.2) lib/rack/handler/webrick.rb:60:in `service'
  /Users/katie/.rvm/rubies/ruby-2.0.0-p247/lib/ruby/2.0.0/webrick/httpserver.rb:138:in `service'
  /Users/katie/.rvm/rubies/ruby-2.0.0-p247/lib/ruby/2.0.0/webrick/httpserver.rb:94:in `run'
  /Users/katie/.rvm/rubies/ruby-2.0.0-p247/lib/ruby/2.0.0/webrick/server.rb:295:in `block in start_thread'


I, [2013-11-15T21:33:33.889944 #28303]  INFO -- : Started GET "/" for 127.0.0.1 at 2013-11-15 21:33:33 -0600
I, [2013-11-15T21:33:33.916805 #28303]  INFO -- : Processing by WelcomeController#index as HTML
I, [2013-11-15T21:33:33.938053 #28303]  INFO -- :   Rendered welcome/index.html.erb within layouts/application (0.5ms)
I, [2013-11-15T21:33:33.939353 #28303]  INFO -- : Completed 200 OK in 22ms (Views: 5.0ms | ActiveRecord: 0.0ms)
I, [2013-11-15T21:33:33.954841 #28303]  INFO -- : Started GET "/stylesheets/application.css" for 127.0.0.1 at 2013-11-15 21:33:33 -0600
F, [2013-11-15T21:33:33.958073 #28303] FATAL -- : 
ActionController::RoutingError (No route matches [GET] "/stylesheets/application.css"):
  actionpack (4.0.1) lib/action_dispatch/middleware/debug_exceptions.rb:21:in `call'
  actionpack (4.0.1) lib/action_dispatch/middleware/show_exceptions.rb:30:in `call'
  railties (4.0.1) lib/rails/rack/logger.rb:38:in `call_app'
  railties (4.0.1) lib/rails/rack/logger.rb:20:in `block in call'
  activesupport (4.0.1) lib/active_support/tagged_logging.rb:67:in `block in tagged'
  activesupport (4.0.1) lib/active_support/tagged_logging.rb:25:in `tagged'
  activesupport (4.0.1) lib/active_support/tagged_logging.rb:67:in `tagged'
  railties (4.0.1) lib/rails/rack/logger.rb:20:in `call'
  actionpack (4.0.1) lib/action_dispatch/middleware/request_id.rb:21:in `call'
  rack (1.5.2) lib/rack/methodoverride.rb:21:in `call'
  rack (1.5.2) lib/rack/runtime.rb:17:in `call'
  activesupport (4.0.1) lib/active_support/cache/strategy/local_cache.rb:83:in `call'
  rack (1.5.2) lib/rack/sendfile.rb:112:in `call'
  railties (4.0.1) lib/rails/engine.rb:511:in `call'
  railties (4.0.1) lib/rails/application.rb:97:in `call'
  rack (1.5.2) lib/rack/lock.rb:17:in `call'
  rack (1.5.2) lib/rack/content_length.rb:14:in `call'
  rack (1.5.2) lib/rack/handler/webrick.rb:60:in `service'
  /Users/katie/.rvm/rubies/ruby-2.0.0-p247/lib/ruby/2.0.0/webrick/httpserver.rb:138:in `service'
  /Users/katie/.rvm/rubies/ruby-2.0.0-p247/lib/ruby/2.0.0/webrick/httpserver.rb:94:in `run'
  /Users/katie/.rvm/rubies/ruby-2.0.0-p247/lib/ruby/2.0.0/webrick/server.rb:295:in `block in start_thread'


I, [2013-11-15T21:33:33.960228 #28303]  INFO -- : Started GET "/javascripts/application.js" for 127.0.0.1 at 2013-11-15 21:33:33 -0600
F, [2013-11-15T21:33:33.962244 #28303] FATAL -- : 
ActionController::RoutingError (No route matches [GET] "/javascripts/application.js"):
  actionpack (4.0.1) lib/action_dispatch/middleware/debug_exceptions.rb:21:in `call'
  actionpack (4.0.1) lib/action_dispatch/middleware/show_exceptions.rb:30:in `call'
  railties (4.0.1) lib/rails/rack/logger.rb:38:in `call_app'
  railties (4.0.1) lib/rails/rack/logger.rb:20:in `block in call'
  activesupport (4.0.1) lib/active_support/tagged_logging.rb:67:in `block in tagged'
  activesupport (4.0.1) lib/active_support/tagged_logging.rb:25:in `tagged'
  activesupport (4.0.1) lib/active_support/tagged_logging.rb:67:in `tagged'
  railties (4.0.1) lib/rails/rack/logger.rb:20:in `call'
  actionpack (4.0.1) lib/action_dispatch/middleware/request_id.rb:21:in `call'
  rack (1.5.2) lib/rack/methodoverride.rb:21:in `call'
  rack (1.5.2) lib/rack/runtime.rb:17:in `call'
  activesupport (4.0.1) lib/active_support/cache/strategy/local_cache.rb:83:in `call'
  rack (1.5.2) lib/rack/sendfile.rb:112:in `call'
  railties (4.0.1) lib/rails/engine.rb:511:in `call'
  railties (4.0.1) lib/rails/application.rb:97:in `call'
  rack (1.5.2) lib/rack/lock.rb:17:in `call'
  rack (1.5.2) lib/rack/content_length.rb:14:in `call'
  rack (1.5.2) lib/rack/handler/webrick.rb:60:in `service'
  /Users/katie/.rvm/rubies/ruby-2.0.0-p247/lib/ruby/2.0.0/webrick/httpserver.rb:138:in `service'
  /Users/katie/.rvm/rubies/ruby-2.0.0-p247/lib/ruby/2.0.0/webrick/httpserver.rb:94:in `run'
  /Users/katie/.rvm/rubies/ruby-2.0.0-p247/lib/ruby/2.0.0/webrick/server.rb:295:in `block in start_thread'


